docs/explanations/how_to_interpret_unattended_upgrades_is_running.md

```diff
@@ -0,0 +1,81 @@
+# How to interpret if unattended upgrades is running
```

Comment:
I find this title to be slightly confusing. I'm not sure if it should be "How to tell if unattended-upgrades is running" or "How to interpret the output of unattended-upgrades".

Reply:
I think the latter should be better. Changing it now